packages/playground/src/domains/agent-builder/components/agent-edit/agent-profile/tools.tsx

```
@@ -1,133 +1,90 @@
-import type { CSSProperties, ReactNode } from 'react';
-import { useState } from 'react';
-import { useFormContext } from 'react-hook-form';
-import { useAgentColor } from '../../../contexts/agent-color-context';
-import type { AgentBuilderEditFormValues } from '../../../schemas';
+import { useMemo, useState } from 'react';
+import { useAllProviderTools } from '../../../../tool-providers/hooks/use-all-provider-tools';
+import { useToolProviders } from '../../../../tool-providers/hooks/use-tool-providers';
 import type { AgentTool } from '../../../types/agent-tool';
-import { AgentSearchbar } from '../agent-searchbar';
-import { AgentSelectableCard } from '../agent-selectable-card';
+import { TwoPanePickerSkeleton } from './two-pane-picker-skeleton';
```

ESLint

[error] `./two-pane-picker-skeleton` import should occur after import of `./toolkit-filter-pane`

(import/order)